streams/src/main/java/org/apache/kafka/streams/kstream/internals/KTableImpl.java

```diff
@@ -244,6 +246,23 @@ public V apply(Change<V> change) {
         return new KStreamImpl<>(topology, name, sourceNodes);
     }
 
+    @Override
+    public <K1> KStream<K1, V> toStream(KTableKeyMapper<K, K1> kTableKeyMapper) {
+        String toStreamName = topology.newName(TOSTREAM_NAME);
```

Comment:
As for implementation, we could just call toStream().selectKey(), which will add two processors instead of one, but maybe more illustrative on the topology.